.gitmodules

```diff
@@ -16,11 +16,11 @@
 	url = https://github.com/KhronosGroup/glslang.git
 	ignore = dirty
 [submodule "Vulkan/spirv-tools"]
-	path = Vulkan/spirv-tools
+	path = 3rdparty/SPIRV/spirv-tools
```

Comment:
It bugs me that the SPIRV folder is all caps but spirv-tools is all lowercase.
It should either be SPIRV/SPIRV-Tools or spirv/spirv-tools. I'm leaning towards the former for consistency with upstream.